### ISSUE TITLE
Build on Windows phone 8.1 Device fail

### ISSUE THREAD

**Issue by ghost** [brody2consult/cordova-sqlite-storage#494]
Hello,

First, thanks for this plugin, I was using it for my iOS and Android project without any problem.
But for the Windows platform I have to stuggle to make it all work.

I use Visual Studio 2015 and the Cordova CLI for building and running the app.
When I build with the param archs="x86", it works and I can run it on the simulator. But when I want to try to build and deploy on real device with the param archs="arm", Visual studio return this error : 
Error: APPX170: The file 'C:\Users\Lucas\Documents\workspace\demoSQLITE\platforms\windows\ARM\Debug\SQLite3.WindowsPhone\SQLite3.winmd' is not a winmd file, or an error occurred when loading it.  CordovaApp.Phone    C:\Program Files (x86)\MSBuild\Microsoft\VisualStudio\v14.0\AppxPackage\Microsoft.AppXPackage.Targets

This a clean new created project to demonstrate the error. (My real project is a Ionic project).

When I look into my project structure the ARM\Debug\SQLite3.WindowsPhone\ folder exist but it is empty.
I think there are PATH errors when I tried to install/build this plugin.

Thanks for your help.

**Comment by brody2consult** [comment]
When in doubt please try the Visual Studio 2015 GUI to build. To build from the Cordova CLI you need to specify the Windows version, architecture, and perhaps whether or not you target Windows Phone. I will improve the documentation when I get a chance.

**Comment by ghost** [comment]
Hi, thanks for your response.
Visual studio 2015 GUI and Cordova CLI both don't work.

What I've done:
In the console, 
"cordova create test"
"cd test"
"cordova platform add windows"
At this step, if I build the app (VS2015 GUI and Corcoa CLI), it works
"cordova plugin add cordova-sqlite-storage"

And now I can't build. In VS2015 this message is displayed: 
MSB3030 Could not copy the file "C:\Users\Lucas\Documents\workspace\test2\platforms\windows\Debug\SQLite3.WindowsPhone\SQLite3.winmd" because it was not found. 

I put in attachment a copy of the project, if you have time to take a quick look.
In the platforms/windows you can see a build/phone/Debug directory is created but with empty folder/files inside.
I think there is a problem during the plugin install process. (I've tried on 2 differents Windows10 computer with VS2015)

[test2.zip](https://github.com/litehelpers/Cordova-sqlite-storage/files/305645/test2.zip)

**Comment by ghost** [comment]
Sorry the build folder has nothing to do with the problem. Anyway the required files seems to not be copied inside the directory. Is there a way to "manually" install the plugin?

When I go at the root directory / plugin/Cordoa-SQLite-storage all the file for the sql3 library are here,

**Comment by brody2consult** [comment]
In the Visual Studio 2015 GUI did you specify which target CPU architecture?

**Comment by ghost** [comment]
Yes, x86 when I try to run on simulator and ARM when I try to run on device.

**Comment by brody2consult** [comment]
I will try running your test project sometime next week. Unfortunately I am busy with a very critical project right now.

**Comment by ghost** [comment]
Thanks,
I continu to investigate by my side, I will come back if I find the probleme.
Good luck on your project.

**Comment by ghost** [comment]
I have resolved my issue.
I was openning the wrong generated project in my VS, and the first build has to be done via the cordova CLI. It's weird because I had issue just with this plugin.
I publish the correct order for the steps, in case of someone encounters the same issue.

In the cordova CLI:
cordova create myProject
cordova plugin add cordova-sqlite-storage
cordova platform add windows
cordova build windows -- --appx=8.1-phone --archs="x86 arm"

Next in the windows platform, open CordovaApp.sln in Visual Studio.
In the solution explorer right click on "CordovaApp.Phone" -> Set as StartUp Project
Don't forget to select the processor architecture and then you can build + run the project.

Thanks for your help

**Comment by paredesivan** [comment]
@brodybits the plugin is compatible with wp8? or should I use the legacy version ?

**Comment by brody2consult** [comment]
On Jun 13, 2016 7:15 PM, "paredesivan" notifications@github.com wrote:

> @brodybits the plugin is compatible with wp8? or should I use the legacy
> version ?

Please use the legacy version for WP8. This is already documented.
